components/rmrk/Create/CreateCollection.vue

```diff
@@ -1,269 +1,201 @@
-          @click="submit"
+    <BaseCollectionForm v-bind.sync="base" protectiveMargin>
+      <template v-slot:header>
+        <b-field>
+          <div>
+            {{ $t('computed id') }}: <b>{{ rmrkId }}</b>
+          </div>
+        </b-field>
+      </template>
+      <template v-slot:main>
+        <BasicSwitch v-model="unlimited" label="mint.unlimited" />
+        <b-field
+          v-if="!unlimited"
+          class="mt-1"
+          :label="$i18n.t('Maximum NFTs in collection')">
+          <b-numberinput
+            v-model="max"
+            placeholder="1 is minumum"
+            :min="1"></b-numberinput>
+        </b-field>
+        <BasicInput
+          v-model="symbol"
+          :label="$t('mint.collection.symbol.label')"
+          :message="$t('mint.collection.symbol.message')"
+          :placeholder="$t('mint.collection.symbol.placeholder')"
+          @keydown.native.space.prevent
+          maxlength="10"
+          expanded />
+      </template>
+
+      <template v-slot:footer>
+        <SubmitButton
+          label="create collection"
           :disabled="disabled"
           :loading="isLoading"
-          outlined>
-          {{ $t('create collection') }}
-        </b-button>
-      </b-field>
-      <b-field>
-        <Support v-model="hasSupport" :price="filePrice">
-          <template v-slot:tooltip>
-            <Tooltip
-              :label="$t('support.tooltip')"
-              iconsize="is-small"
-              buttonsize="is-small"
-              tooltipsize="is-medium" />
-          </template>
-        </Support>
-      </b-field>
-    </div>
+          @click="submit" />
+      </template>
+    </BaseCollectionForm>
   </div>
 </template>
 
 <script lang="ts">
 import { Component, mixins } from 'nuxt-property-decorator'
-import { emptyObject } from '@/utils/empty'
-
 import Connector from '@vue-polkadot/vue-api'
-import exec, { execResultValue, txCb } from '@/utils/transactionExecutor'
 import { notificationTypes, showNotification } from '@/utils/notification'
-import SubscribeMixin from '@/utils/mixins/subscribeMixin'
 import RmrkVersionMixin from '@/utils/mixins/rmrkVersionMixin'
-import { Collection, CollectionMetadata } from '../service/scheme'
 import { unSanitizeIpfsUrl } from '@/utils/ipfs'
-import { decodeAddress } from '@polkadot/keyring'
-import { u8aToHex } from '@polkadot/util'
 import { generateId } from '@/components/rmrk/service/Consolidator'
 import { supportTx, calculateCost } from '@/utils/support'
-import NFTUtils from '../service/NftUtils'
-import TransactionMixin from '@/utils/mixins/txMixin'
+import MetaTransactionMixin from '@/utils/mixins/metaMixin'
 import { pinFileToIPFS, pinJson, PinningKey } from '@/utils/pinning'
+import AuthMixin from '~/utils/mixins/authMixin'
+import {
+  createMetadata,
+  createCollection,
+  CreatedCollection,
+  createMintInteaction,
+  Interaction,
+  asSystemRemark,
+  addressToHex,
+} from '@vue-polkadot/minimark'
```

Comment:
New phone, who's this?

Reply:
[minimark](https://github.com/vikiival/minimark)